### DISCUSSION TITLE
Beatmap title with special charcters has different order than in Stable

### DISCUSSION THREAD

**Discussion by PercyDan54** [ppy/osu#24107] [Q&A]
### Type

Game behaviour

### Bug description

In stable, if a beatmap has special characters in its title, it will be at the top of beatmap listing, above beatmaps starting with numbers (sorting with Title)

But lazer places them at the bottom (below beatmaps starting with Z)

### Screenshots or videos
Note the order of beatmap "[A]ddiction"

![osu_2023-07-02_13-54-39](https://github.com/ppy/osu/assets/50285552/8fc2c5e3-1d0e-49c9-b43d-81d5105efec3)

![screenshot-070](https://github.com/ppy/osu/assets/50285552/bd462bb9-ff45-4e0f-a647-edb388bc0177)

### Version

2023.621.0

### Logs

Not required

**Comment by peppy** [comment]
Not sure which makes more sense, of that there should be any guarantee. For whatever it's worth, apple puts symbols at the end.
